### PR TITLE
fix($browser): Darken primary color plus small a11y tweaks

### PR DESCRIPTION
**Pull request by mwickett**
Darkened the primary colors, added aria-label to locale selector and created unique IDs for logo SVG
paths

**What**: Primary colors have been changed for accessibility (AA) compliance. Also added unique ID attributes to Glamorous logo SVG path elements and added an `aria-label` attribute to the locale selector.

**Why**: Based on #23 I ran the site through a number of `a11y` checkers (primarily using [aXe](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd)) and the color contrast ratios weren't high enough.

**How**:  I darkened the primary and primaryMed color while slightly lightening the primaryLight color.

screenshot 2017-06-10 21 04 45

Additional issues were:

1. The locale selector was unlabelled. I added an `aria-label`
2. The Glamorous logo SVG paths all had `id="Shape"` which isn't AA compliant, so I created unique, description IDs

I fully realize that the color changes do alter the design - I've tried my best to maintain the same feel while acheiving AA compliance. 

There are two outstanding issues:

1. The small lipstick link back to `/` needs text (which can be hidden) - haven't been able to dig into that link styling deep enough yet. Also, I've never used 💄 yet :)
2. The home page code example section has contrast issues - I couldn't figure out where the Prism style settings are configured - point me there and I'll take a crack.

### CODE REVIEW

**Comment by kentcdodds** [comment]
This is fantastic! Thank you very much for the contributions! I'll give this a closer look later, but I wanted to just say that this is great!

cc @jonmajorc

**Comment by paulmolluzzo** [comment]
Restarting the Travis build. It failed on the deployment, not the actual tests. Those pass locally for me.

**Comment by paulmolluzzo** [comment]
@kentcdodds this is the relevant error and is not really a reflection on the quality of this PR I think.

```
[deploy]   throw new Error('Missing required environment variable GH_TOKEN')
[deploy]   ^
[deploy] 
[deploy] Error: Missing required environment variable GH_TOKEN
[deploy]     at Object.<anonymous> (/home/travis/build/kentcdodds/glamorous-website/other/now-travis:16:9)
[deploy]     at Module._compile (module.js:569:30)
[deploy]     at Object.Module._extensions..js (module.js:580:10)
[deploy]     at Module.load (module.js:503:32)
[deploy]     at tryModuleLoad (module.js:466:12)
[deploy]     at Function.Module._load (module.js:458:3)
[deploy]     at Function.Module.runMain (module.js:605:10)
[deploy]     at startup (bootstrap_node.js:158:16)
[deploy]     at bootstrap_node.js:575:3
```

File: examples/home-page-example.js [hunk 1]
```diff
@@ -15,13 +15,13 @@ export default locale => {
     })
 
     const solidColors = {
-      backgroundColor: '#ED5C70',
+      backgroundColor: '#DA233C',
```
Comment (paulmolluzzo) [inline examples/home-page-example.js]: I kind of prefer `#CC3A4B` here. Is that still acceptable from an a11y perspective?

Comment (paulmolluzzo) [inline examples/home-page-example.js]: This is the new `primaryMed` by the way.

Comment (mwickett) [inline examples/home-page-example.js]: That works, good call.

**Comment by mwickett** [comment]
@paulmolluzzo You mentioned the code snippet, and I agree. I couldn't figure out how to update the prism syntax highlighting colors. Happy to update if you can point me in the right direction.

**Comment by mwickett** [comment]
Also, I failed to notice that the Travis build failed - hopefully not something I did incorrectly. If so, please let me know!

**Comment by paulmolluzzo** [comment]
> @paulmolluzzo You mentioned the code snippet, and I agree. I couldn't figure out how to update the prism syntax highlighting colors. Happy to update if you can point me in the right direction.

I'm not referring to the react live area, I mean the install snippet below that:

screen shot 2017-06-10 at 10 49 17 pm

> Also, I failed to notice that the Travis build failed - hopefully not something I did incorrectly. If so, please let me know!

It's not from your code. 😉

**Comment by mwickett** [comment]
@paulmolluzzo I definitely see what you mean. With the more saturated bg color, that grey doesn't work.

What about shifting to `colors.blue` - that would match `pre` elements across the site and the really dark blue looks pretty good. 

screenshot 2017-06-11 13 06 37

**Comment by paulmolluzzo** [comment]
@mwickett that's better, yeah! Can you change that in this PR please?

**Comment by mwickett** [comment]
Made that change @paulmolluzzo and also updated CTA button color to primaryMed instead of primary, for consistency. Think it looks better. Now, primary is so dark (because it has to be for text) that primaryMed is a more useful color in large quantities (bg color etc.)

Also added "Home" to the `/` 💄 icon link in the nav, and hid the text visually with a negative margin. 

I created a `Hidden` component using Glamorous (first time I've actually used it!), hope it's ok there. Thought it might be useful higher up in case it's needed elsewhere, but I'll look for guidance there.

```
// Used to hide text for a11y - ie. Home in navigation
const Hidden = glamorous.span({
  position: 'absolute',
  left: '-10000px',
  top: 'auto',
  width: '1px',
  height: '1px',
  overflow: 'hidden',
})
```

I believe the only outstanding issue now is the Prism syntax highlighting colors on the front page. Where do those live right now? I'm obviously missing something because I can't figure out where that's being set.

**Comment by paulmolluzzo** [comment]
> Now, primary is so dark (because it has to be for text) that primaryMed is a more useful color in large quantities (bg color etc.)

Maybe we want to use that for the huge BG color on the home page then? 🤔

As for overriding the Prism styles, look here for more info: https://github.com/FormidableLabs/react-live/blob/master/README.md#liveeditor-

**Comment by mwickett** [comment]
@paulmolluzzo It is pretty intense at that scale!

It's [already using `primaryMed`](https://github.com/kentcdodds/glamorous-website/blob/master/pages/index/index.js#L21). It could go to a custom, less intense color, but then I worry about losing the consistency.

Thanks for the link, now I understand how that's being rendered. Going to dig into it now.

**Comment by kentcdodds** [comment]
Let's push forward with this. I'm going to go ahead and merge this and we can fix anything that's not quite right in future PRs. Thanks for a great review @paulmolluzzo and thank you so much @mwickett for this extremely valuable contribution! :clap:

**Pull request merged**